guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc

```diff
@@ -59,6 +65,25 @@ euid = __ID_of_Apache_User__
 # chown root.{apache-user} /etc/httpd/conf/http.keytab
 # chmod 640 /etc/httpd/conf/http.keytab
 ----
+. Configure SSSD to map Group Policy Objects (GPOs) from AD to the `foreman` PAM service:
+.. Add the following lines to your `/etc/sssd/sssd.conf`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+access_provider = ad
+ad_gpo_access_control = enforcing
+ad_gpo_map_service = +foreman
+----
+ifdef::satellite[]
++
+For more information on GPOs, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/integrating_rhel_systems_directly_with_windows_active_directory/managing-direct-connections-to-ad_integrating-rhel-systems-directly-with-active-directory#applying-group-policy-object-access-control-in-rhel_managing-direct-connections-to-ad[How SSSD interprets GPO access control rules] in _Integrating RHEL systems directly with Windows Active Directory_.
+endif::[]
+.. Restart the `sssd` service:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# systemctl restart sssd
+----
```

Comment:
Moved from the now deleted WARNING box for better visibility. Also, added a step to restart SSSD.